core/src/main/codegen/templates/Parser.jj

```diff
@@ -5379,6 +5381,34 @@ SqlCall JsonLengthFunctionCall() :
     }
 }
 
+SqlCall JsonRemoveFunctionCall() :
+{
+    final List<SqlNode> elements = new ArrayList<SqlNode>();
+    final SqlNode[] otherArgs = new SqlNode[1];
```

Comment:
I think the variable names such as `elements`, `otherArgs` don't make sense. Could you please improve a bit? Since the JSON_REMOVE's arg definition is different from others. E.g. `jsonDoc`, `pathExprs`.

Reply:
well,I'll change it.